src/main/groovy/nextflow/executor/AwsBatchExecutor.groovy

```diff
@@ -371,7 +376,7 @@ class AwsBatchFileCopyStrategy extends SimpleFileCopyStrategy {
      */
     @Override
     String touchFile( Path file ) {
-        "touch ${file.name} && nxf_s3_upload ${file.name} s3:/${file} && rm ${file.name}"
+        "touch ${file.name} && nxf_s3_upload ${file.name} s3:/${file.getParent()} && rm ${file.name}"
     }
```

Comment:
This and the ones below should not change because these are always supposed to be single files.

Reply:
The getParent() method is called even on single files since we need to retrieve the full path of the working directory to store the file on S3